kubernetes/mlflow-tracking-operator-master/Gopkg.lock

```diff
@@ -0,0 +1,921 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
are all of these vendor packages required for `mlflow`? Can this list be reduced to only required dependencies.

Alternative, if this file is autogenerated, can we skip adding this to repo and create as required?

Reply:
This file can be deleted as I believe this gets generated. I'll remove that.